src/components/Block/Results.vue

```diff
@@ -33,45 +37,64 @@ const choices = computed(() =>
     :title="ts >= proposal.end ? $t('results') : $t('currentResults')"
   >
     <div v-for="choice in choices" :key="choice.i">
-      <div class="link-color mb-1">
-        <span
-          v-tippy="{
-            content: choice.choice.length > 12 ? choice.choice : null
-          }"
-          class="mr-1"
-          v-text="_shorten(choice.choice, 'choice')"
-        />
+      <template v-if="!(proposal.type === 'basic' && hideAbstain && choice.i === 2)">
```

Comment:
changes here summarized:
- template tag wrapper to hide abstain option
- additional versions (for, against) of the last span element (the percentage)
- different max value for progress bar

Reply:
Cool, I didn't know about using `template` to conditional multiple tags without using a parent tag in vue.